src/app/frontend/overview/workloadstatus/template.html

```diff
@@ -14,132 +14,124 @@
-<!--<div layout="column"-->
-<!--layout-align="center center">-->
-<!--<kd-allocated-resources-chart color-palette="::$ctrl.colorPalette"-->
-<!--data="::$ctrl.resourcesRatio.replicaSetRatio"-->
-<!--ratio="0"-->
-<!--enable-tooltips="true"-->
-<!--size="140">-->
-<!--</kd-allocated-resources-chart>-->
-<!--<div class="kd-graph-title">-->
-<!--[[Replica Sets|Label 'Replica Sets' for the resource status chart on overview page.]]-->
-<!--</div>-->
-<!--</div>-->
-<!--</div>-->
+    <div class="kd-graph-container"
+         *ngIf="resourcesRatio.jobRatio.length">
+      <div fxLayout="column"
+           fxLayoutAlign="center center">
+        <kd-allocation-chart [id]="'jobs'"
+                             [colorPalette]="colors"
+                             [data]="resourcesRatio.jobRatio"
+                             [ratio]="0"
+                             [enableTooltips]="true"
+                             [size]="140"></kd-allocation-chart>
+      </div>
+      <div class="kd-graph-title">Jobs</div>
+    </div>
 
-<!--<div class="kd-graph-container"-->
-<!--ng-if="::$ctrl.resourcesRatio.rcRatio.length">-->
-<!--<div layout="column"-->
-<!--layout-align="center center">-->
-<!--<kd-allocated-resources-chart color-palette="::$ctrl.colorPalette"-->
-<!--data="::$ctrl.resourcesRatio.rcRatio"-->
-<!--ratio="0"-->
-<!--enable-tooltips="true"-->
-<!--size="140">-->
-<!--</kd-allocated-resources-chart>-->
-<!--<div class="kd-graph-title">-->
-<!--[[Replication Controllers|Label 'RCs' for the resource status chart on overview page.]]-->
-<!--</div>-->
-<!--</div>-->
-<!--</div>-->
+    <div class="kd-graph-container"
+         *ngIf="resourcesRatio.podRatio.length">
+      <div fxLayout="column"
+           fxLayoutAlign="center center">
+        <kd-allocation-chart [id]="'pods'"
+                             [colorPalette]="colors"
+                             [data]="resourcesRatio.podRatio"
+                             [ratio]="0"
+                             [enableTooltips]="true"
+                             [size]="140"></kd-allocation-chart>
+      </div>
+      <div class="kd-graph-title">Pods</div>
+    </div>
 
-<!--<div class="kd-graph-container"-->
-<!--ng-if="::$ctrl.resourcesRatio.statefulSetRatio.length">-->
-<!--<div layout="column"-->
-<!--layout-align="center center">-->
-<!--<kd-allocated-resources-chart color-palette="::$ctrl.colorPalette"-->
-<!--data="::$ctrl.resourcesRatio.statefulSetRatio"-->
-<!--ratio="0"-->
-<!--enable-tooltips="true"-->
-<!--size="140">-->
-<!--</kd-allocated-resources-chart>-->
-<!--<div class="kd-graph-title">-->
-<!--[[Stateful Sets|Label 'Stateful Sets' for the resource status chart on overview page.]]-->
-<!--</div>-->
-<!--</div>-->
-<!--</div>-->
-<!--</div>-->
+    <div class="kd-graph-container"
+         *ngIf="resourcesRatio.replicaSetRatio.length">
+      <div fxLayout="column"
+           fxLayoutAlign="center center">
+        <kd-allocation-chart [id]="'replicaSets'"
```

Comment:
This could be just `id="replicaSets"` if you are not using binding anyway. Same applies to other resources.